downstream/modules/hub/proc-add-user-to-group.adoc

```diff
@@ -15,8 +14,8 @@ Complete the following steps to add users to an existing group.
 
 . Log in to your local {HubName}.
 . From the navigation panel, select menu:User Access[Groups].
-. Click on a Group name.
-. Navigate to the menu:Users[] tab, then click btn:[Add].
+. Click on a *Group* name.
+. Navigate to the *menu:*Users* tab and click btn:[Add].
 . Select users to add from the list and click btn:[Add].
 
-When you add the selected users to the group, they have permissions to use {HubName} assigned to the group.
+When you add the selected users to the group, they have the permissions to use {HubName} that are assigned to the group.
```

Comment:
It looks like {HubName} evaluates to "automation hub". If that's the case the sentence reads "they have the permissions to use automation hub that are assigned to the group." Should automation hub be automation hubs?

Reply:
Yes the phrasing is awkward here, but it refers to users having the "permissions" assigned to that group in hub. I think it should be rephrased for clarity.

Reply:
Ah thanks for clarifying 👍 agreed!